### ISSUE TITLE
Enable keeping duplicated geometries, but marking as "duplicated" with a property

### ISSUE THREAD

**Issue by robyngit** [PermafrostDiscoveryGateway/viz-staging#9]
For the IWP dataset, we decided to _keep_ geometries that are classified as "duplicates" during processing in the version of the geopackage tiles that we archive. We will, however, mark duplicate geometries with a "duplicates" property. We therefore need to update the deduplication method to optionally keep duplicated geometries, but set a new "duplicated" property to True.

Duplicates should still be removed during the creation of the web products (3d tiles, PNG web tiles)

**Comment by robyngit** [comment]
I am going to start working on this. I think we should restructure things a little so that:
1) If there is a deduplication method set in the config, then duplicates are always identified during the staging process
2) The deduplicate_at config option will be interpreted as the step(s) at which the duplicates should be _removed_ (staging, raster, and/or 3dtiles)

**Comment by robyngit** [comment]
This feature is working in the `feature-9-flag-duplicates` branch. I've tested with both the neighbor and footprints method, but we will need to make updates to other parts of the workflow before we can implement this:
- [ ] Update viz-raster to remove polygons flagged as duplicates, rather than run the deduplication method again
- [ ] Update the StagedTo3DConverter to remove polygons flagged as duplicates, rather than run the deduplication method again
- [ ] Update the merge script in the ray workflow to flag duplicates with the new methods when two staged tiles are merged

**Comment by robyngit** [comment]
The new deduplication strategy is now merged into the main branch of all relevant packages. Briefly, with the new strategy:
- Duplicate polygons are always identified during the staging step (and during the merging step, in the case that we are staging across multiple nodes and merging later)
- During staging, a new boolean attribute (default name: `staging_duplicated`) is added that identifies polygons as duplicates or not
- The `deduplicate_at` config option is interpreted to mean the step(s) at which polygons flagged as duplicates should be *removed*
